wunderio/next-drupal-starterkit
# Update npm dependencies

Updates all npm dependencies to the very latest versions available, except:

- Next.js and related dependencies are still on v13 even though v14 exists. This is because Next.js 14 requires a minimum of node 18.17.0 or node 20.0.0, but we need to use similar node versions everywhere we use them (lando, circleci, silta). It should be possible to just update to node 20 once there's a node 20 image available in silta-cicd images.
- TypeScript updated to the latest version supported by typescript-eslint

To test:
- Checkout this branch locally and confirm that the site features still work with no errors in browser devtools or next.js console. Be sure to test login, search, menus, etc. You'll need to run commands like:
  - `lando npm i && lando npm run lint && lando npm run build && lando npm run start`
  - `lando npm run dev`
  - `lando npm run storybook`
  - `lando npm run cypress:run`
